resources/localization/de/PrusaSlicer_de.po

```diff
@@ -5,15 +5,28 @@ msgstr ""
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
-"X-Generator: PhraseApp (phraseapp.com)\n"
+"X-Generator: Poedit 2.2.4\n"
+"Project-Id-Version: \n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: \n"
+"Last-Translator: \n"
+"Language-Team: \n"
 
 #: src/slic3r/GUI/PresetHints.cpp:39
-msgid "\nDuring the other layers, fan"
-msgstr "\nWährend der übrigen Schichten, Lüfter"
+msgid ""
+"\n"
+"During the other layers, fan"
+msgstr ""
+"\n"
+"Während der übrigen Schichten, Lüfter"
 
 #: src/slic3r/GUI/PresetHints.cpp:35
-msgid "\nIf estimated layer time is greater, but still below ~%1%s, fan will run at a proportionally decreasing speed between %2%%% and %3%%%."
-msgstr "\nFalls die erwartete Schichtdruckzeit größer, aber noch unterhalb von  ~%1%s ist,  wird der Lüfter mit einer sich proportional verringernden  Geschwindigkeit zwischen %2%%% und %3%%% laufen."
```

Comment:
Here are 2x two spaces (between "ist,  wird" and "verringernden  Geschwindigkeit"), may be not intended.

Reply:
Oh, and between "von  ~%1%s"